src/binder_nfc_adapter.c

```diff
@@ -242,8 +246,10 @@ binder_nfc_callback_handle_data(
     if (data && gbinder_reader_at_end(reader)) {
         NciHalClient* hal_client = self->hal_client;
 
-        DUMP("%c data, %u byte(s)", DIR_IN, (guint)len);
+#ifndef _NO_NFC_VERBOSE_
+        gutil_log(&binder_hexdump_log, GLOG_LEVEL_VERBOSE, "%c data, %u byte(s)", DIR_IN, (guint)len);
         binder_dump_data(DIR_IN, data, len);
+#endif /* _NO_NFC_VERBOSE_ */
```

Comment:
Please define one macro at the top and use it

Reply:
Thanks for the review. I have combined that parts together.